core/src/main/java/tech/pegasys/eth2signer/core/service/http/handlers/SignForIdentifierHandler.java

```diff
@@ -27,23 +28,48 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+import org.hyperledger.besu.plugin.services.metrics.Counter;
 
 public class SignForIdentifierHandler implements Handler<RoutingContext> {
   private static final Logger LOG = LogManager.getLogger();
   private final SignerForIdentifier<?> signerForIdentifier;
+  private final Counter totalCounter;
+  private final Counter missingSignerCounter;
+  private final Counter malformedRequestCounter;
 
-  public SignForIdentifierHandler(final SignerForIdentifier<?> signerForIdentifier) {
+  public SignForIdentifierHandler(
+      final SignerForIdentifier<?> signerForIdentifier,
+      final MetricsSystem metrics,
+      final String metricsPrefix) {
     this.signerForIdentifier = signerForIdentifier;
+    totalCounter =
```

Comment:
Thinking we should remove the existing http metrics potentially? with the way the urls are constructed we will have histogram metric for each public key which could be in the thousands